### PR TITLE
MM-61272 - Scrolled to scheduled post without error

### PR DESCRIPTION
**Pull request by harshilsharma63**
#### Summary
Scrolled to scheduled post without error

#### Ticket Link
Fixes https://mattermost.atlassian.net/browse/MM-61272

#### Release Note
```release-note
none
```

### CODE REVIEW

File: webapp/channels/src/components/drafts/scheduled_post_list/scheduled_post_list.tsx
Comment (Rajat-Dabade) [inline webapp/channels/src/components/drafts/scheduled_post_list/scheduled_post_list.tsx:72]: nit: `posst` -> `post`

**Review by Rajat-Dabade — APPROVED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]

**Review by yasserfaraazkhan — APPROVED** [review]

**Comment by harshilsharma63** [comment]
This can be merged into master once the feature branch is merged into master as well

**Pull request merged**
